UltraStar Play/Assets/Scenes/Sing/SingSceneController.cs

```diff
@@ -166,6 +165,33 @@ void Start()
         LayoutRebuilder.ForceRebuildLayoutImmediate(CanvasUtils.FindCanvas().GetComponent<RectTransform>());
     }
 
+    private void InitLyricsDisplayer(LyricsDisplayer lyricsDisplayer)
+    {
+        bool needSecondLyricsDisplayer = SongMeta.GetVoices().Count > 1 && PlayerControllers.Count > 1;
+        if (lyricsDisplayer.isTop)
```

Comment:
this bool is never changed... Did you maybe forget adding some files to the changelist before committing?

Reply:
The boolean flag was pretty stupid (it was set in the Inspector). I only wanted to differentiate the instances.
I now added references to the instances in the SingSceneController, which also makes FindObjectsOfType unecessary.